docker-fedora/init.sh

```diff
@@ -1,15 +1,13 @@
-#!/bin/bash
+#!/bin/sh
 
 # set storage first
 (
-    . /etc/sysconfig/docker-storage-setup
-    /usr/bin/docker-storage-setup
+        . /etc/sysconfig/docker-storage-setup
```

Comment:
just want the indent to match all over.

You might want to reorganize this content so all of the docker-* can share the same content.